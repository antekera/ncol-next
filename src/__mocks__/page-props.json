{
  "pageProps": {
    "preview": false,
    "pageTitle": "Mundial de Qatar 2022: Se dio inicio a la fiesta mundialista",
    "pageType": "/SINGLE",
    "post": {
      "title": "Mundial de Qatar 2022: Se dio inicio a la fiesta mundialista",
      "antetituloNoticia": "Mundial de Qatar 2022: Se dio inicio a la fiesta mundialista",
      "fuenteNoticia": "Mundial de Qatar 2022: Se dio inicio a la fiesta mundialista",
      "slug": "inicio-la-inauguracion-del-mundial-de-qatar-2022",
      "date": "2022-11-20T10:57:15",
      "excerpt": "<p>La espera se terminó: el Mundial Qatar 2022 se puso en marcha con una ceremonia inaugural a pura música y color en el estadio Al Bayt. Más de 60 mil espectadores",
      "featuredImage": {
        "node": {
          "sourceUrl": "https://i0.wp.com/admin.noticiascol.com/wp-content/uploads/2022/11/Screenshot_9-8.jpg?fit=616%2C347&ssl=1",
          "altText": "",
          "caption": null
        }
      },
      "categories": {
        "edges": [
          {
            "node": {
              "name": "_Pos_Destacado",
              "uri": "/category/_pos_destacado/",
              "slug": "_pos_destacado"
            }
          },
          {
            "node": {
              "name": "Copa Mundial Catar 2022",
              "uri": "/category/deportes/copa-mundial-catar-2022/",
              "slug": "copa-mundial-catar-2022"
            }
          },
          {
            "node": {
              "name": "Deportes",
              "uri": "/category/deportes/",
              "slug": "deportes"
            }
          },
          {
            "node": {
              "name": "Futbol",
              "uri": "/category/deportes/futbol/",
              "slug": "futbol"
            }
          }
        ]
      },
      "tags": { "edges": [] },
      "contentType": { "node": { "id": "cG9zdF90eXBlOnBvc3Q=" } },
      "isPreview": false,
      "isRestricted": false,
      "isRevision": false,
      "status": "publish",
      "template": { "templateName": "Default" },
      "uri": "/2022/11/20/inicio-la-inauguracion-del-mundial-de-qatar-2022/",
      "content": "<p>La espera se terminó: el Mundial Qatar 2022 se puso en marcha con una ceremonia inaugural a pura música y color en el estadio Al Bayt. Más de 60 mil espectadores disfrutaron del show de apertura de la cita máxima y observaron de cerca la Copa del Mundo, que fue exhibida por el francés Marcel Desailly. Además, se movieron al ritmo de Jung Kook, líder de la banda surcoreana BTS, y escucharon atentamente las palabras de Morgan Freeman y el jeque Tamim bin Hamad Al Thani.</p>\n<h2 id=\"lbp_a1\">La&#8217;eeb voló por el cielo de Qatar 2022.</h2>\n<p>La mascota oficial del Mundial, fiel representación del mundo árabe, dijo presente en el inicio de la ceremonia.</p>\n<img loading=\"lazy\" class=\"aligncenter\" src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506990/inauguracion-mundial_416x234.webp?resize=567%2C319&#038;ssl=1\" alt=\"Inauguración Mundial\" width=\"567\" height=\"319\" data-recalc-dims=\"1\" />\n<h2 id=\"lbp_a2\">El desfile de las mascotas históricas del Mundial.</h2>\n<p>En la inauguración de Qatar 2022, pasearon por el campo de juego de Al Bay los viejos personajes representativos de las Copas del Mundo.</p>\n<img loading=\"lazy\" class=\"aligncenter\" src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506987/desfile-mascotas-mundial_416x234.webp?resize=485%2C273&#038;ssl=1\" alt=\"desfile mascotas Mundial\" width=\"485\" height=\"273\" data-recalc-dims=\"1\" />\n<div class=\"lbp-item\">\n<h2 id=\"lbp_a3\">Jung Kook, de BTS, presente en la inauguración del Mundial Qatar 2022.</h2>\n<p>El vocalista de la popular banda surcoreana le puso música y color al arranque de la Copa del Mundo el Al Bayt.</p>\n<p>&nbsp;</p>\n<figure class=\"rat16x9\"><img loading=\"lazy\" class=\"\" title=\"BTS Mundial Qatar\" src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_416x234.webp?resize=453%2C255&#038;ssl=1\" srcset=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_416x234.webp?resize=453%2C255&#038;ssl=1 480w, https://media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_862x485.webp 1080w\" alt=\"BTS Mundial Qatar\" width=\"453\" height=\"255\" data-src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_416x234.webp?resize=453%2C255&#038;ssl=1\" data-srcset=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_416x234.webp?resize=453%2C255&#038;ssl=1 480w, https://media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_862x485.webp 1080w\" data-recalc-dims=\"1\" /></figure>\n</div>\n<div class=\"lbp-item\">\n<h2 id=\"lbp_a4\">La Copa del Mundo, en buenas manos.</h2>\n<p>El francés Marcel Desailly, campeón del mundo en Francia 1998, le mostró el trofeo que se disputará en el Mundial de Qatar 2022 a la multitud presente para la apertura y el primer partido, entre el local y Ecuador.</p>\n<p>&nbsp;</p>\n<figure class=\"rat16x9\"><img title=\"Marcel Desailly\" src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506981/marcel-desailly_416x234.webp?ssl=1\" srcset=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506981/marcel-desailly_416x234.webp?ssl=1 480w, https://media.tycsports.com/files/2022/11/20/506981/marcel-desailly_862x485.webp?v=1 1080w\" alt=\"Marcel Desailly\" data-src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506981/marcel-desailly_416x234.webp?ssl=1\" data-srcset=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506981/marcel-desailly_416x234.webp?ssl=1 480w, https://media.tycsports.com/files/2022/11/20/506981/marcel-desailly_862x485.webp?v=1 1080w\" data-recalc-dims=\"1\" /></figure>\n</div>\n<div class=\"lbp-item\">\n<p>&nbsp;</p>\n</div>\n",
      "customFields": {
        "antetituloNoticia": "lorem ipsum",
        "fuenteNoticia": "tycsports.com",
        "videoNoticia": null
      }
    },
    "content": [
      "<p>La espera se terminó: el Mundial Qatar 2022 se puso en marcha con una ceremonia inaugural a pura música y color en el estadio Al Bayt. Más de 60 mil espectadores disfrutaron del show de apertura de la cita máxima y observaron de cerca la Copa del Mundo, que fue exhibida por el francés Marcel Desailly. Además, se movieron al ritmo de Jung Kook, líder de la banda surcoreana BTS, y escucharon atentamente las palabras de Morgan Freeman y el jeque Tamim bin Hamad Al Thani.</p>",
      "<p\n<h2 id=\"lbp_a1\">La&#8217;eeb voló por el cielo de Qatar 2022.</h2>\n<p>La mascota oficial del Mundial, fiel representación del mundo árabe, dijo presente en el inicio de la ceremonia.</p>\n<img loading=\"lazy\" class=\"aligncenter\" src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506990/inauguracion-mundial_416x234.webp?resize=567%2C319&#038;ssl=1\" alt=\"Inauguración Mundial\" width=\"567\" height=\"319\" data-recalc-dims=\"1\" />\n<h2 id=\"lbp_a2\">El desfile de las mascotas históricas del Mundial.</h2>\n<p>En la inauguración de Qatar 2022, pasearon por el campo de juego de Al Bay los viejos personajes representativos de las Copas del Mundo.</p>\n<img loading=\"lazy\" class=\"aligncenter\" src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506987/desfile-mascotas-mundial_416x234.webp?resize=485%2C273&#038;ssl=1\" alt=\"desfile mascotas Mundial\" width=\"485\" height=\"273\" data-recalc-dims=\"1\" />\n<div class=\"lbp-item\">\n<h2 id=\"lbp_a3\">Jung Kook, de BTS, presente en la inauguración del Mundial Qatar 2022.</h2>\n<p>El vocalista de la popular banda surcoreana le puso música y color al arranque de la Copa del Mundo el Al Bayt.</p>\n\n<figure class=\"rat16x9\"><img loading=\"lazy\" class=\"\" title=\"BTS Mundial Qatar\" src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_416x234.webp?resize=453%2C255&#038;ssl=1\" srcset=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_416x234.webp?resize=453%2C255&#038;ssl=1 480w, https://media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_862x485.webp 1080w\" alt=\"BTS Mundial Qatar\" width=\"453\" height=\"255\" data-src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_416x234.webp?resize=453%2C255&#038;ssl=1\" data-srcset=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_416x234.webp?resize=453%2C255&#038;ssl=1 480w, https://media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_862x485.webp 1080w\" data-recalc-dims=\"1\" /></figure>\n</div>\n<div class=\"lbp-item\">\n<h2 id=\"lbp_a4\">La Copa del Mundo, en buenas manos.</h2>\n<p>El francés Marcel Desailly, campeón del mundo en Francia 1998, le mostró el trofeo que se disputará en el Mundial de Qatar 2022 a la multitud presente para la apertura y el primer partido, entre el local y Ecuador.</p>\n\n<figure class=\"rat16x9\"><img title=\"Marcel Desailly\" src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506981/marcel-desailly_416x234.webp?ssl=1\" srcset=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506981/marcel-desailly_416x234.webp?ssl=1 480w, https://media.tycsports.com/files/2022/11/20/506981/marcel-desailly_862x485.webp?v=1 1080w\" alt=\"Marcel Desailly\" data-src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506981/marcel-desailly_416x234.webp?ssl=1\" data-srcset=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506981/marcel-desailly_416x234.webp?ssl=1 480w, https://media.tycsports.com/files/2022/11/20/506981/marcel-desailly_862x485.webp?v=1 1080w\" data-recalc-dims=\"1\" /></figure>\n</div>\n<div class=\"lbp-item\">\n\n</div>\n"
    ],
    "posts": {
      "edges": [
        {
          "node": {
            "excerpt": "<p>La espera se terminó: el Mundial Qatar 2022 se puso en marcha con una ceremonia inaugural a pura música y color en el estadio Al Bayt. Más de 60 mil espectadores",
            "title": "Brasil: De varios disparos asesinan a venezolano.(video)",
            "slug": "brasil-de-varios-disparos-asesinan-a-venezolano-video",
            "date": "2022-11-20T09:45:41",
            "featuredImage": {
              "node": {
                "sourceUrl": "https://i0.wp.com/admin.noticiascol.com/wp-content/uploads/2022/11/Screenshot_8-10.jpg?fit=386%2C224&ssl=1",
                "altText": "",
                "caption": null
              }
            },
            "categories": {
              "edges": [
                {
                  "node": {
                    "name": "_Pos_Columna_izq",
                    "uri": "/category/_pos_columna_izq/",
                    "slug": "_pos_columna_izq"
                  }
                },
                {
                  "node": {
                    "name": "Internacionales",
                    "uri": "/category/internacionales/",
                    "slug": "internacionales"
                  }
                },
                {
                  "node": {
                    "name": "Sucesos",
                    "uri": "/category/sucesos/",
                    "slug": "sucesos"
                  }
                }
              ]
            },
            "tags": { "edges": [] },
            "contentType": { "node": { "id": "cG9zdF90eXBlOnBvc3Q=" } },
            "isPreview": false,
            "isRestricted": false,
            "isRevision": false,
            "status": "publish",
            "template": { "templateName": "Default" },
            "uri": "/2022/11/20/brasil-de-varios-disparos-asesinan-a-venezolano-video/"
          }
        },
        {
          "node": {
            "title": "Municipio Maracaibo: Accidente de transito deja como saldo un motorizado muerto",
            "slug": "municipio-maracaibo-accidente-de-transito-deja-como-saldo-un-motorizado-muerto",
            "date": "2022-11-20T09:39:55",
            "featuredImage": {
              "node": {
                "sourceUrl": "https://i0.wp.com/admin.noticiascol.com/wp-content/uploads/2022/07/morgue-hospial-de-maracaibo-2.jpg?fit=700%2C400&ssl=1",
                "altText": "",
                "caption": null
              }
            },
            "categories": {
              "edges": [
                {
                  "node": {
                    "name": "_Pos_Columna_izq",
                    "uri": "/category/_pos_columna_izq/",
                    "slug": "_pos_columna_izq"
                  }
                },
                {
                  "node": {
                    "name": "Maracaibo",
                    "uri": "/category/zulia/maracaibo/",
                    "slug": "maracaibo"
                  }
                },
                {
                  "node": {
                    "name": "Sucesos",
                    "uri": "/category/sucesos/",
                    "slug": "sucesos"
                  }
                }
              ]
            },
            "tags": { "edges": [] },
            "contentType": { "node": { "id": "cG9zdF90eXBlOnBvc3Q=" } },
            "isPreview": false,
            "isRestricted": false,
            "isRevision": false,
            "status": "publish",
            "template": { "templateName": "Default" },
            "uri": "/2022/11/20/municipio-maracaibo-accidente-de-transito-deja-como-saldo-un-motorizado-muerto/"
          }
        }
      ]
    },
    "ads": {
      "menu": {
        "id": "div-gpt-ad-1662938702901-0",
        "style": { "minWidth": "1000px", "minHeight": "250px" }
      },
      "menu_mobile": {
        "id": "div-gpt-ad-1663287871941-0",
        "style": { "minWidth": "300px", "minHeight": "100px" }
      },
      "sidebar": {
        "id": "div-gpt-ad-1662933959123-0",
        "style": { "minWidth": "300px", "minHeight": "600px" }
      },
      "sidebar2": {
        "id": "div-gpt-ad-1662937413122-0",
        "style": { "minWidth": "300px", "minHeight": "250px" }
      },
      "cover": {
        "id": "div-gpt-ad-1663288350704-0",
        "style": { "minWidth": "300px", "minHeight": "100px" }
      },
      "homeFeed": {
        "id": "div-gpt-ad-1662935751586-0",
        "style": { "minWidth": "300px", "minHeight": "100px" }
      },
      "homeFeed2": {
        "id": "div-gpt-ad-1662935053350-0",
        "style": { "minWidth": "200px", "minHeight": "200px" }
      },
      "homeFeed3": {
        "id": "div-gpt-ad-1663438486397-0",
        "style": { "minWidth": "200px", "minHeight": "200px" }
      },
      "categoryFeed": {
        "id": "div-gpt-ad-1663289413827-0",
        "style": { "minWidth": "300px", "minHeight": "100px" }
      },
      "categoryFeed2": {
        "id": "div-gpt-ad-1663289569751-0",
        "style": { "minWidth": "300px", "minHeight": "100px" }
      },
      "squareC1": {
        "id": "div-gpt-ad-1663290647151-0",
        "style": { "minWidth": "300px", "minHeight": "250px" }
      },
      "squareC2": {
        "id": "div-gpt-ad-1663290682160-0",
        "style": { "minWidth": "300px", "minHeight": "250px" }
      },
      "squareC3": {
        "id": "div-gpt-ad-1663290699468-0",
        "style": { "minWidth": "300px", "minHeight": "250px" }
      }
    }
  }
}
