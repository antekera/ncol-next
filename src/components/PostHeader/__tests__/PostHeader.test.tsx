import React from 'react'

import { render } from '@testing-library/react'

import { PostHeader } from '..'
import { pageProps } from '../../../__mocks__/page-props.json'

jest.mock('next/router', () => ({
  useRouter() {
    return {
      route: '/',
      pathname: '',
      query: '',
      asPath: '',
      push: jest.fn(),
      events: {
        on: jest.fn(),
        off: jest.fn()
      },
      beforePopState: jest.fn(() => null),
      prefetch: jest.fn(() => null)
    }
  }
}))

describe('PostHeader', () => {
  test('should match snapshots', () => {
    const { container } = render(<PostHeader {...pageProps.post} />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
