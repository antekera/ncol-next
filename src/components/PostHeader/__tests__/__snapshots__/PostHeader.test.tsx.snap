// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostHeader should match snapshots 1`] = `
<div
  class="container mx-auto px-6 sm:px-7"
>
  <a
    aria-label="Copa Mundial Catar 2022"
    class="link-post-category relative mr-2 inline-block font-sans text-xs leading-none top-6 ml-1 rounded border-secondary bg-primary px-1 pb-[3px] pt-1 uppercase text-white hover:border-solid hover:bg-slate-400 hover:text-white"
    href="/categoria/copa-mundial-catar-2022"
  >
    Copa Mundial Catar 2022
  </a>
  <a
    aria-label="Deportes"
    class="link-post-category relative mr-2 inline-block font-sans text-xs leading-none top-6 ml-1 rounded border-secondary bg-primary px-1 pb-[3px] pt-1 uppercase text-white hover:border-solid hover:bg-slate-400 hover:text-white"
    href="/categoria/deportes"
  >
    Deportes
  </a>
  <a
    aria-label="Futbol"
    class="link-post-category relative mr-2 inline-block font-sans text-xs leading-none top-6 ml-1 rounded border-secondary bg-primary px-1 pb-[3px] pt-1 uppercase text-white hover:border-solid hover:bg-slate-400 hover:text-white"
    href="/categoria/futbol"
  >
    Futbol
  </a>
  <h1
    class="mb-4 mt-10 font-sans_medium text-2xl leading-tight text-slate-700 sm:w-11/12 sm:text-3xl md:mb-10 md:mt-12 md:text-5xl lg:text-5xl"
  >
    Mundial de Qatar 2022: Se dio inicio a la fiesta mundialista
  </h1>
  <p
    class="-mt-6 mb-6 pt-3 font-sans leading-6 text-slate-500 sm:w-11/12 md:-mt-6 md:mb-7 md:text-xl lg:text-2xl"
  >
    Mundial de Qatar 2022: Se dio inicio a la fiesta mundialista
  </p>
  <div
    class="w-full border-t border-solid border-slate-200 pb-2 pt-4 font-sans text-sm text-slate-500 md:flex md:justify-between"
  >
    <div
      class="pr-2"
    >
      <time>
        <span
          class="capitalize"
        >
          noviembre 20, 2022
        </span>
        <span>
           • 10:57 a.m.
        </span>
      </time>
      <div
        class="inline"
      >
        <span
          class="pl-2 pr-3"
        >
          |
        </span>
        <span>
          Con información de 
        </span>
        Mundial de Qatar 2022: Se dio inicio a la fiesta mundialista
      </div>
    </div>
    <div
      class="hidden pt-2 md:block md:pt-0"
    >
      <span
        class="mr-3 hidden font-sans sm:inline md:mr-4"
      >
        Compártelo
      </span>
      <div
        class="has-tooltip mr-3 inline-block h-4 w-5 md:mr-4"
      >
        <span
          class="invisible shadow-sm tooltip absolute z-10 -ml-4 mt-1 whitespace-nowrap rounded bg-gray-200 p-1 px-2 py-1 text-sm text-primary"
        >
          ¡Enlace copiado!
        </span>
        <a
          class="z-1 relative relative hover:text-primary"
          href="#"
          rel="noreferrer noopener"
          title="Copia el enlace"
        >
          <svg
            class="lucide lucide-link"
            fill="none"
            height="20"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"
            />
            <path
              d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"
            />
          </svg>
        </a>
      </div>
      <a
        class="mr-3 inline-block h-4 w-4 hover:text-primary md:mr-4"
        href="https://www.facebook.com/sharer.php?u=https://noticiascol.comundefined"
        rel="noreferrer noopener"
        target="_blank"
        title="Compartir en Facebook"
      >
        <svg
          aria-hidden="true"
          class="w-5"
          data-icon="facebook"
          data-prefix="fab"
          focusable="false"
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M279.14 288l14.22-92.66h-88.91v-60.13c0-25.35 12.42-50.06 52.24-50.06h40.42V6.26S260.43 0 225.36 0c-73.22 0-121.08 44.38-121.08 124.72v70.62H22.89V288h81.39v224h100.17V288z"
            fill="currentColor"
          />
        </svg>
      </a>
      <a
        class="mr-3 inline-block h-4 w-5 hover:text-primary md:mr-4"
        href="https://twitter.com/intent/tweet?url=https://noticiascol.comundefined"
        rel="noreferrer noopener"
        target="_blank"
        title="Compartir en X"
      >
        <svg
          aria-hidden="true"
          class="w-5"
          data-icon="x"
          data-prefix="fab"
          focusable="false"
          role="img"
          viewBox="0 0 322 380"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M178.57 127.15 290.27 0h-26.46l-97.03 110.38L89.34 0H0l117.13 166.93L0 300.25h26.46l102.4-116.59 81.8 116.59h89.34M36.01 19.54H76.66l187.13 262.13h-40.66"
            fill="currentColor"
          />
        </svg>
      </a>
      <a
        class="mr-3 inline-block h-4 w-5 hover:text-primary md:mr-4"
        data-action="share/whatsapp/share"
        href="whatsapp://send?text=Mira esta noticia publicada en noticiascol.com: https://noticiascol.comundefined"
        title="Compartir por WhatsApp"
      >
        <svg
          aria-hidden="true"
          class="w-5"
          data-icon="whatsapp"
          data-prefix="fab"
          focusable="false"
          role="img"
          viewBox="0 0 26 26"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M.057 24l1.687-6.163c-1.041-1.804-1.588-3.849-1.587-5.946.003-6.556 5.338-11.891 11.893-11.891 3.181.001 6.167 1.24 8.413 3.488 2.245 2.248 3.481 5.236 3.48 8.414-.003 6.557-5.338 11.892-11.893 11.892-1.99-.001-3.951-.5-5.688-1.448l-6.305 1.654zm6.597-3.807c1.676.995 3.276 1.591 5.392 1.592 5.448 0 9.886-4.434 9.889-9.885.002-5.462-4.415-9.89-9.881-9.892-5.452 0-9.887 4.434-9.889 9.884-.001 2.225.651 3.891 1.746 5.634l-.999 3.648 3.742-.981zm11.387-5.464c-.074-.124-.272-.198-.57-.347-.297-.149-1.758-.868-2.031-.967-.272-.099-.47-.149-.669.149-.198.297-.768.967-.941 1.165-.173.198-.347.223-.644.074-.297-.149-1.255-.462-2.39-1.475-.883-.788-1.48-1.761-1.653-2.059-.173-.297-.018-.458.13-.606.134-.133.297-.347.446-.521.151-.172.2-.296.3-.495.099-.198.05-.372-.025-.521-.075-.148-.669-1.611-.916-2.206-.242-.579-.487-.501-.669-.51l-.57-.01c-.198 0-.52.074-.792.372s-1.04 1.016-1.04 2.479 1.065 2.876 1.213 3.074c.149.198 2.095 3.2 5.076 4.487.709.306 1.263.489 1.694.626.712.226 1.36.194 1.872.118.571-.085 1.758-.719 2.006-1.413.248-.695.248-1.29.173-1.414z"
            fill="currentColor"
          />
        </svg>
      </a>
      <a
        class="relative top-1 mr-3 block inline-block w-5 hover:text-primary md:mr-4"
        href="#comentarios"
        title="Ver los comentarios"
      >
        <svg
          class="lucide lucide-message-circle-more"
          fill="none"
          height="22"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="22"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M7.9 20A9 9 0 1 0 4 16.1L2 22Z"
          />
          <path
            d="M8 12h.01"
          />
          <path
            d="M12 12h.01"
          />
          <path
            d="M16 12h.01"
          />
        </svg>
      </a>
    </div>
  </div>
</div>
`;
